lib/block_view_dao_coin_limit_order_test.go

```diff
@@ -1647,10 +1652,11 @@ func TestDAOCoinLimitOrder(t *testing.T) {
 		_, _, _, _, err = _connectDAOCoinLimitOrderTxn(
 			testMeta, m1Pub, m1Priv, currentTxn, totalInputMake)
 		require.Error(err)
-		require.Contains(err.Error(), RuleErrorDAOCoinLimitOrderFeeNanosOverflow)
+		require.Contains(err.Error(), RuleErrorDAOCoinLimitOrderFeeNanosBelowMinTxFee)
```

Comment:
This is the better error message when someone tries to set FeeNanos to zero $DESO so this is a good change IMO.